onnxruntime/core/providers/cuda/tensor/split.cc

```diff
@@ -13,27 +13,36 @@ ONNX_OPERATOR_VERSIONED_KERNEL_EX(Split,
                                   2, 10,
                                   kCudaExecutionProvider,
                                   (*KernelDefBuilder::Create()).TypeConstraint("T", DataTypeImpl::AllFixedSizeTensorTypes()),
-                                  Split);
+                                  Split_2_13);
```

Comment:
did you mean 1_13?

Reply:
The CUDA kernel registration starts at opset 2 not 1. Not sure of the history but was just matching that with the name.